File: archive/Source/2019/Day1_1.exs
fuelRequired = fn x -> (div x, 3) - 2 end

IO.puts(
File.stream!("Day1.txt", [], :line)
    |> Enum.map(&String.trim(&1))
    |> Enum.filter(&(&1 != ""))
    |> Enum.map(&fuelRequired.(String.to_integer(&1)))
    |> Enum.sum
)

File: archive/Source/2020/Day1_1.py

file = open("Day1.txt", "r")

data = []
for line in file:
    data.append(int(line[:-1]))

for d1 in range(len(data)-1):
    for d2 in range(d1+1, len(data)):
        if (data[d1] + data[d2] == 2020):
            print (data[d1] * data[d2])
